db/migrate/20151106130955_update_miq_shortcuts.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateMiqShortcuts < ActiveRecord::Migration
+  def up
+    MiqShortcut.find_each do |row|
```

Comment:
- Please wrap in `say_with_time` as with other data migrations
- Don't use a real model in a migration.  Always create a stub.
- Create a migration spec.